app/models/user.rb

```diff
@@ -14,6 +14,13 @@ class User < ActiveRecord::Base
   has_and_belongs_to_many :courses
   has_many :schedules, dependent: :destroy
 
+  # relationships for textbook listings
+  has_many :textbook_listings, :class_name => 'TextbookTransaction', :foreign_key => 'seller_id'
```

Comment:
you don't need string, just :class_name => TextbookTransaction without quotes work fine